From 2c50e17244cad02944a0ad417b8da6776f5f9d6a Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 06 Nov 2019 12:58:15 +0200
Subject: [PATCH] [SW-0] hl: remove Library path from Makefile

Seem that toolchain manage to locate static libgcc without this help

Change-Id: I363f719fde3e38d62a7d7430b60642794597ecce
---

diff --git a/Makefile b/Makefile
index 2da177a..8236064 100644
--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,6 @@
 
 LD_SCRIPT := $(HL_DEV)/preboot.lds
 LDFLAGS := -g -T $(LD_SCRIPT) -nostdlib -static
-LDFLAGS += -L ../../host/opt/ext-toolchain/lib/gcc/aarch64-linux-gnu/6.4.1
 LDLIBS := -lgcc
 
 include $(HL_DEV)/Makefile
